trigger_attestation.md

```diff
@@ -0,0 +1,448 @@
+challenging, as they require an adversary to have a deep understanding of the
+reporting origin’s key space in order to alter aggregate results. A much more
+practical attack is one closer to replaying honest reports, since it is
+fairly easy for an adversary to learn a set of honest reports via instrumented
+web browsing, and to replicate them.
+
+## Blind signatures overview
+
+At a high level, [Private State
+Tokens](https://developer.chrome.com/docs/privacy-sandbox/trust-tokens/)
+implements a [blind
+signature](https://en.wikipedia.org/wiki/Blind_signature)-style operation:
+
+```
+BlindMessage := Blind(message)
+
+BlindSignature := Sign(blindedMessage)
+
+ClearSignature := Unblind(blindSignature)
+
+Verify(message, ClearSignature) => boolean
+```
+
+In this document, we use the following terms:
+* **Blinded token**: Equivalent to the `BlindMessage`
+* **Token**: Equivalent to the `(message, ClearSignature)` pair
+
+For simplicity, this omits the [public/private
+metadata](https://github.com/WICG/trust-token-api#extension-public-metadata)
+aspects of Private State Tokens which extend this basic primitive.
+
+
+## Reporting origin opt-in
+
+In order for a reporting origin to use this feature, they must configure
+themselves as a Private State Token issuer, by publishing key commitments
+publicly (see [issue 117](https://github.com/WICG/trust-token-api/issues/117) on
+the Private State Token repo, as this process is still being finalized).
+
+
+## API changes
+
+```mermaid
+sequenceDiagram
+    participant B as Browser
+    participant A as Adtech
+    B->>A: Source ping
+    A->>B: Source registration
+    Note over A,B: Time passes
+    B->>A: Trigger ping, bearing blinded id/dest
+    A->>A: Check IVT filter
+    A->>B: Trigger registration, with blind-signed id/dest
+    B->>B: Verify signature and unblind id/dest + signature
+    Note over A,B: Time passes
+    B->>A: Send report alongside signature
+```
+
+### Issuance
+
+The token issuance protocol can only happen on [trigger
+registration](https://github.com/WICG/attribution-reporting-api/blob/main/AGGREGATE.md#attribution-trigger-registration).
+For reporting origins that are configured to do Private State Token issuance,
+all new requests that have the `Attribution-Reporting-Eligible` request header
+with the `trigger` value will automatically attach a
+`Sec-Attribution-Reporting-Private-State-Token` header with a blinded message
+(which embeds concatenation of the `report_id` and `attribution_destination`).
+
+The reporting origin will inspect the request and decide whether it is valid,
+i.e. whether the origin suspects it is coming from a real, honest client and
+should therefore be allowed to generate attribution reports.
+
+*   If the request is considered **invalid** and hence shouldn’t be taken into
+    account to calculate aggregate measurement results, the reporting origin
+    should respond without adding a
```

Comment:
nit: this conflicts with what the IVT detector stuff that we call out further below, might be worth adding a callout to that section from here, but this is probably fine as-is